docs/Behaviour - Server Side.md

```diff
@@ -83,6 +83,8 @@ The `/properties` endpoint shows properties of the Input. In particular, it MAY
 
 ### Base EDID
 
+An Input MAY support adjusting Base EDID to its internal capabilities via `adjust_to_caps` query parameter of a `PUT /edid/base` request.
```

Comment:
Please clarify this .... is the base EDID modified on PUT such that a Get of the base EDID would return the modified base EDID ... or without active constraint the base EDID is modified to become the effective EDID ... etc ...